src/cf-forms.less

```diff
@@ -448,6 +456,102 @@ textarea {
 
 }
 
+/* topdoc
+  name: Range input
+  family: cf-forms
+  patterns:
+    - name: Range slider input
+      markup: |
+        <label for="range-slider">Range Slider</label>
+        <input id="range-slider" type="range" min="0" max="100">
+  tags:
+    - cf-forms
+*/
+
+input[type=range] {
+    width: 100%;
+    padding-left: 0;
+    margin: 15px 0;
+    -webkit-appearance: none;
+    border: none;
+    &::-webkit-slider-runnable-track {
```

Comment:
It looks like `auto-prefixer` won't support this [until there's an unprefixed spec](https://github.com/postcss/autoprefixer/issues/279) available. Should we make this and the thumb a mixin that auto generates all the necessary prefixed versions rather than having to make multiple changes if things change?